check.py

```diff
@@ -195,21 +195,14 @@ def fail_if_not_contained(actual, expected):
           raise Exception('wasm interpreter error: ' + err) # failed to pretty-print
         raise Exception('wasm interpreter error')
 
-print '\n[ checking binaryen-shell... ]\n'
```

Comment:
Correct, it's not tested elsewhere. It just seemed arbitrary to test it just for the shell, it seems better to test all the bin/ content or none of it. I can add such a test if you want.